gitops/README.md

```diff
@@ -0,0 +1,65 @@
+# Initialize the GitOps environment
+## Install the operators
+The `Red Hat OpenShift Pipelines` and `Red Hat OpenShift GitOps` operators can be installed from the solution 
+derived from the [Janus IDP Demo](https://github.com/redhat-gpte-devopsautomation/janus-idp-bootstrap)
```

Comment:
What about rewording like:
"[...] operators can be installed using the solution derived from the Janus IDP Demo [...]"